src/site/content/en/blog/aria-poison-or-antidote/index.md

```diff
@@ -0,0 +1,283 @@
+Firefox vs. Chrome, it's often our bug, since Firefox has an older, more seasoned accessibility
+mapping layer.
+
+If all the browser - screen reader combinations seem to agree, and the web content still behaves
+badly, we will need to file a bug on the web content itself, not on the browser.
+
+## Summary
+
+In summary, ARIA magic can be used to override anything and everything that the HTML says, or add to
+it. It can be used to do little fine changes to the accessibility presentation, or to create an
+entire experience. This is why ARIA is both incredibly powerful and yet dangerous in the hands of
+our friendly local web authors that don't generally use screen readers themselves. 
+
+ARIA is just a dumb truth override markup layer. When a screen reader asks what's happening, if ARIA
+exists, they get the ARIA version of the truth instead of the real underlying truth.
+
+## Addendum 1: Additional Resources
+
+### Hybrid reference with keyboard info and code examples
+
++   [W3C's ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.1/): this
+    documents the important keyboard navigation characteristics of each example and provides
+    working JS/CSS/ARIA code. The examples are focused on what works today, and do not cover mobile.
+
+## Addendum 2: What is ARIA most used for?
+
+Because ARIA can be used to replace or supplement small or large truths, it's kind of a generally
+useful thing for saying stuff that the screen reader cares about. 
+
+Here are some common uses of ARIA.
+
+1.  Special widgets that don't exist in HTML, like a menubar, autocomplete, tree, or spreadsheet
+1.  Widgets that exist in HTML, but the author invented their own anyway, possibly because they
+    needed to tweak the behavior or appearance of the normal widget. For example, an HTML `<input
+    type="range">` element is basically a slider, but authors want to make it look different. For
+    most things, CSS can be used, but for `input type="range"`, CSS is very awkward to use. We can
+    make our own slider, and use `role="slider"` on it with `aria-valuenow` to say what the current
+    value is.
+1.  Live regions tell screen readers "in this area of the page, anything that changes is worth telling
+    the user about."
+1.  Landmarks (HTML has equivalents now). These are somewhat like headings, in that they help
+    screen reader users find what they want faster. However, they're different in that they contain
+    the entire related area. Like, "this container is the main area of the page" and "this
+    container over here is a navigation panel". 
+
+## Addendum 3: What's an Accessibility API?
+
+An accessibility API is how a screen reader or other AT knows what's in the page and what's
+happening right now. Examples are MSAA, IA2 and UIA. And that's just Windows! There are two parts to
+an accessibility API:
+
++   A "tree" of objects that represents a container hierarchy. Like Russian nesting dolls,
```

Comment:
```suggestion
+   A "tree" of objects that represents a container hierarchy. These are like Russian nesting dolls,
```

Reply:
Bullets can start with sentence fragments. Within a single bullet, every statement after the first must be a complete sentence.